lib/Dialect/TritonGPU/Transforms/WarpSpecialization/PartitionLoops.cpp

```diff
@@ -516,6 +517,75 @@ LogicalResult triton::gpu::partitionLoop(scf::ForOp loop) {
   return success();
 }
 
+LogicalResult inferIfStmtPartitions(scf::IfOp ifOp) {
```

Comment:
`inferIfOpPartitions`, to be consistent with `inferReduceOpPartitions`

Reply:
done